commands/storage-provider/update/mod.ts

```diff
@@ -0,0 +1,20 @@
+import { command } from "../../../zcli.ts";
+import { s3 } from "./s3/mod.ts";
+
+/**
+ * This variable is automatically generated by `zcli add`. Do not remove this
+ * or change its name unless you're no longer using `zcli add`.
+ */
+const subCommands: ReturnType<typeof command>[] = [s3];
+
+export const update = command("update", {
+  short: "Update a storage provider",
+  long: `
+    Update a storage provider.
+  `,
+  commands: subCommands,
+}).run(function* ({ ctx }) {
+  for (const line of update.help(ctx)) {
+    yield line;
+  }
```

Comment:
Does the update only updates s3? or why is this one returning a message and we have `update/s3`?